plasma_framework/contracts/src/vaults/verifiers/EthDepositVerifier.sol

```diff
@@ -10,9 +10,14 @@ import {PaymentOutputModel as DepositOutputModel} from "../../transactions/outpu
 contract EthDepositVerifier is IEthDepositVerifier {
     using DepositOutputModel for DepositOutputModel.Output;
 
+    uint256 public supportedOutputType;
     // Hardcoded transaction type for payment transaction
     uint8 constant internal DEPOSIT_TX_TYPE = 1;
 
+    constructor(uint256 outputType) public {
```

Comment:
Same as above, inject or hardcode both txType and outputType

Reply:
done